### ISSUE TITLE
[BUG] zmodel does not support calling @default(dbgenerated()) with no parameters

### ISSUE THREAD

**Issue by clementoriol** [zenstackhq/zenstack#1400]
**Description and expected behavior**

**Description**
Prisma does not support have first-class support for [Postgres generated columns](https://www.postgresql.org/docs/13/ddl-generated-columns.html) but they still kind of work if you : 

- Write the migration yourself
- Declare them in the prisma schema as optional with `@default(dbgenerated())` [(see the prisma docs)](https://www.prisma.io/docs/orm/reference/prisma-schema-reference#dbgenerated)
```
generatedField    String?                 @default(dbgenerated()) /// note the empty argument
```

**Issue**
However, zmodels don't seem to allow calling dbgenerated() without arguments. Running `zenstack generate` with the previous example will output the following error : 

```
missing argument for parameter "expr"
```

**Expected behavior**
zmodels should allow `@default(dbgenerated())` and generate the prisma schema accordingly

**Environment**

-   ZenStack version: 2.0.2
-   Prisma version: 5.10.2
-   Database type: Postgresql

**Comment by ymc9** [comment]
Thanks for providing the fix @clementoriol !

**Comment by ymc9** [comment]
Fixed in release 2.0.3.
